lib/pacemaker/pcmk_output.c

```diff
@@ -2512,6 +2512,61 @@ ticket_constraints_xml(pcmk__output_t *out, va_list args)
     return pcmk_rc_ok;
 }
 
+PCMK__OUTPUT_ARGS("ticket-state", "xmlNode *")
+static int
+ticket_state_default(pcmk__output_t *out, va_list args)
+{
+    xmlNode *state_xml = va_arg(args, xmlNode *);
+
+    GString *buf = g_string_sized_new(1024);
+
+    pcmk__formatted_printf(out, "State XML:\n\n");
```

Comment:
This can only be called for text output (I missed that the first time around)

Reply:
I think it's fine to just use `out->info()` here - realistically, the default is only going to be used for text.  I suppose it's possible it could be used for none or logging, in which case yeah we don't want error from calling pcmk__formatted_printf.  The constraint message will need the same modification.